lib/MooX/Should.pm

```diff
@@ -2,8 +2,8 @@ package MooX::Should;
 
 # ABSTRACT: optional type restrictions for Moo attributes
 
-use Moo       ();
-use Moo::Role ();
+use Moo 2.003006 ();
```

Comment:
I have placed the version here. Is this fine? When I run `dzil build`, it updates the `cpanfile` with this version as expected.